<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget1{
	background-color: rgb(106, 90, 205)
}

#table{
	background-color: rgb(194, 194, 255)
}


#syncButton{
	background-color: rgb(106, 90, 205);
	border-top-left-radius: 12;
	border-bottom-left-radius: 12;
	color: white
}

#syncButton2{
	background-color: rgb(106, 90, 205);
	border-top-right-radius: 12;
	border-bottom-right-radius: 12;
	color: white;
	border-left-color: rgb(255, 255, 255)
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QWidget" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>29</y>
      <width>541</width>
      <height>341</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>1</y>
       <width>551</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {
    background-color: rgb(255, 255, 255);
	border: none
}</string>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="syncButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>280</y>
       <width>45</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Select files to Sync</string>
     </property>
     <property name="text">
      <string>SYNC
Files</string>
     </property>
    </widget>
    <widget class="QPushButton" name="syncButton2">
     <property name="geometry">
      <rect>
       <x>486</x>
       <y>280</y>
       <width>45</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#syncButton2{
	text : SYNC \n Folder
}</string>
     </property>
     <property name="text">
      <string>SYNC
Folders</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>